invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -169,6 +169,7 @@ def get_form_config(**kwargs):
     return dict(
         vocabularies=vocabularies,
         current_locale=str(current_i18n.locale),
+        default_locale=BABEL_DEFAULT_LOCALE,
```

Comment:
This needs to come from ``current_app.config.get('BABEL_DEFAULT_LOCALE', 'en')`` instead, and I can see it's a bug that was introduced earlier so you probably also have to fix it higher up in the file.

Otherwise instances cannot change the default language.

Reply:
Ah. yes it makes sense, thanks!